gradio/components.py

```diff
@@ -5330,6 +5331,94 @@ def as_example(self, input_data: str | None) -> str:
         return postprocessed if postprocessed else ""
 
 
+@document("change")
+class Code(Changeable, IOComponent, SimpleSerializable):
+    """
+    Creates a Code editor for entering, editing or viewing code.
+    Preprocessing: passes a {str} of code.
+    Postprocessing: expects the function to return a {str} filepath or a {str} of code.
+    """
+
+    languages = [
+        "python",
+        "markdown",
+        "json",
+        "html",
+        "css",
+        "javascript",
+        "typescript",
+        "yaml",
+        "dockerfile",
+        "shell",
+        "r",
+    ]
+
+    def __init__(
+        self,
+        value: str | None = None,
+        language: str | None = None,
+        *,
+        label: str | None = None,
+        show_label: bool = True,
+        visible: bool = True,
+        elem_id: str | None = None,
+        **kwargs,
```

Comment:
Does `Code` support an `interactive` parameter like the other components that can be used both as an input and an output?